addons/arsenal/functions/fnc_scanConfig.sqf

```diff
@@ -125,7 +125,7 @@ private _magazineGroups = [[],[]] call CBA_fnc_hashCreate;
             (_cargo select 17) pushBackUnique _className;
         };
     };
-} foreach configProperties [_configCfgWeapons, "isClass _x && {getNumber (_x >> 'scope') == 2} && {getNumber (_x >> 'ace_arsenal_hide') != 1}", true];
+} foreach configProperties [_configCfgWeapons, "isClass _x && {2 == (if (isNumber (_x >> 'scopeArsenal')) then {getNumber (_x >> 'scopeArsenal')} else {getNumber (_x >> 'scope')})} && {getNumber (_x >> 'ace_arsenal_hide') != 1}", true];
```

Comment:
Why is the `2 == ` in front? That looks so weird.

Reply:
yoda condition 😃

Reply:
I usually use this form, kind `short == very_long_expression`, it's much easier to see what is checked here.

Reply:
As I said though this ignores items with scope!=2 and scopeArsenal==2.
Checking both if ofcause a tiny bit slower (lazy eval) but this runs once at gameStart soo..